loader/pom.xml

```diff
@@ -77,11 +77,18 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-rest</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.mongodb.morphia</groupId>
-            <artifactId>morphia</artifactId>
-            <version>${morphia.version}</version>
-        </dependency>
+
```

Comment:
Doesn't the loader use Morphia anymore??

Reply:
Yes, it does. It's inherited from the top level pom.